QuickSubmitForm.php

```diff
@@ -283,7 +283,7 @@ public function initData()
             $this->_submission = Repo::submission()->dao->newDataObject();
             $this->_submission->setContextId($this->_context->getId());
             $this->_submission->setStatus(PKPSubmission::STATUS_QUEUED);
-            $this->_submission->setSubmissionProgress(1);
+            $this->_submission->setSubmissionProgress('start');
```

Comment:
The `1` translates better to `files`, but I think it's better to start the UI from the beginning if the submission wasn't completed through the QuickSubmit.